.github/workflows/snap.yml

```diff
@@ -0,0 +1,40 @@
+name: Snap
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, ready_for_review]
+
+jobs:
+  Snap:
+    runs-on: ubuntu-latest
+
+    timeout-minutes: 30
+
+    steps:
+    - name: Install Snapcraft
+      uses: samuelmeuli/action-snapcraft@v1
+      with:
+        snapcraft_token: ${{ secrets.SNAPCRAFT_TOKEN }}
+        use_lxd: true
+
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Patch PPA and version
+      run: |
+        sed -i 's@mir-team/release@mir-team/dev@' snap/snapcraft.yaml
+        sed -i 's@rev-list --count@rev-parse --short@' snap/snapcraft.yaml
+
+    - name: Build and verify the snap
+      env:
+        SNAPCRAFT_BUILD_INFO: 1
+        SNAP_ENFORCE_RESQUASHFS: 0
+      run: |
+        sg lxd -c '/snap/bin/snapcraft --use-lxd'
+
+        sudo snap install review-tools
+        /snap/bin/review-tools.snap-review --slots snap/local/slots.json *.snap
+
+    - name: Publish the snap
+      run: |
+        snapcraft upload *.snap --release "edge/pr${{ github.event.number }}"
```

Comment:
This doesn't seem to be a useful designation: I see `edge/pr30` and `edge/pr31`, but this PR is #47

Reply:
I think you confused this with MirServer/mir#31:
```plain
 snapcraft revisions mir-kiosk | grep pr47
5295    2020-10-21T13:21:03Z  amd64   2.1.0+dev74-snap70d8a5e  edge/pr47*
5294    2020-10-21T12:28:49Z  amd64   2.1.0-snap1              edge/pr47
```